web_src/js/features/repo-unicode-escape.ts

```diff
@@ -1,27 +1,28 @@
 import {addDelegatedEventListener, hideElem, queryElemSiblings, showElem, toggleElem} from '../utils/dom.ts';
 
 export function initUnicodeEscapeButton() {
+  // buttons might appear on these pages: file view (code, rendered markdown), diff (commit, pr conversation, pr diff), blame, wiki
   addDelegatedEventListener(document, 'click', '.escape-button, .unescape-button, .toggle-escape-button', (btn, e) => {
     e.preventDefault();
 
-    const fileContentElemId = btn.getAttribute('data-file-content-elem-id');
-    const fileContent = fileContentElemId ?
-      document.querySelector(`#${fileContentElemId}`) :
+    const unicodeContentSelector = btn.getAttribute('data-unicode-content-selector');
+    const container = unicodeContentSelector ?
+      document.querySelector(unicodeContentSelector) :
       btn.closest('.file-content, .non-diff-file-content');
-    const fileView = fileContent?.querySelectorAll('.file-code, .file-view');
+    const fileView = container.querySelector('.file-code, .file-view') || container;
```

Comment:
```suggestion
    const fileView = container.querySelector('.file-code, .file-view') ?? container;
```

Very nitpicky but I prefer to use `??` for fallbacks to values that can not become empty string, 0, etc.

Reply:
Yup, `??` is indeed better